primitives/messages/src/lib.rs

```diff
@@ -69,21 +69,30 @@ impl OperatingMode for MessagesOperatingMode {
 
 /// Lane id which implements `TypeId`.
 #[derive(
-	Clone,
-	Copy,
-	Decode,
-	Default,
-	Encode,
-	Eq,
-	Ord,
-	PartialOrd,
-	PartialEq,
-	RuntimeDebug,
-	TypeInfo,
-	MaxEncodedLen,
+	Clone, Copy, Decode, Default, Encode, Eq, Ord, PartialOrd, PartialEq, TypeInfo, MaxEncodedLen,
 )]
 pub struct LaneId(pub [u8; 4]);
 
+pub use lane_id_debug_impl::*;
+
+#[cfg(not(feature = "std"))]
+mod lane_id_debug_impl {
+	impl core::fmt::Debug for super::LaneId {
+		fn fmt(&self, fmt: &mut core::fmt::Formatter) -> core::fmt::Result {
+			fmt.write_str("<stripped>")
+		}
+	}
+}
+
+#[cfg(feature = "std")]
+mod lane_id_debug_impl {
+	impl std::fmt::Debug for super::LaneId {
+		fn fmt(&self, fmt: &mut std::fmt::Formatter) -> std::fmt::Result {
+			self.0.fmt(fmt)
+		}
+	}
+}
+
```

Comment:
Do we want to print `<stripped>` for non-std environments ? Was this the previous behaviour ?

Could we just do the following ?

```
impl core::fmt::Debug for LaneId {
	fn fmt(&self, fmt: &mut core::fmt::Formatter) -> core::fmt::Result {
		self.0.fmt(fmt)
	}
}
```

Would this work for both std and non-std environments ?

Reply:
No, "<stripped>" is how `RuntimeDebug` works. I think you're right and in previous version, it was printing the same from the no-std env too, because we weren't using some custom type. Let's change it